bundles/org.openhab.ui/web/src/components/widgets/layout/oh-canvas-layout.vue

```diff
@@ -0,0 +1,260 @@
+          :icon-f7="grid.enable ? 'circle_grid_3x3_fill' : 'scircle_grid_3x3'"
+          style="margin-left: auto" />
+        <f7-menu-item
+          @click="
+            context.editmode.configureWidget(
+              context.component,
+              context.parent,
+              'oh-canvas-layout'
+            )
+          "
+          text="Configure" />
+      </f7-menu>
+      <hr>
+    </f7-block>
+    <div
+      ref="canvasLayoutContainer"
+      class="oh-canvas-layout-container"
+      :style="{
+        background: context.editmode
+          ? 'var(--f7-page-master-border-color)'
+          : false,
+        'user-select': context.editmode ? 'none' : 'auto',
+        width: style.width + 'px',
+        height: style.height + 'px',
+        transform: `scale(${style.scale})`,
+        textAlign: 'center',
+        position: 'relative',
+        '--oh-canvas-item-box-shadow': config.boxShadow
+          ? config.boxShadow
+          : '0px 0px 4px 2px #444',
+        '--oh-canvas-item-svg-shadow': config.filterShadow
+          ? config.filterShadow
+          : 'drop-shadow(0px 0px 4px #444)',
+        '--oh-canvas-item-text-shadow': config.textShadow
+          ? config.textShadow
+          : '#444 0px 0px 4px',
+      }">
+      <div
+        style="
+          height: inherit;
+          width: inherit;
+          position: absolute;
+          top: 0;
+          left: 0;
+        ">
+        <img
+          class="oh-canvas-background"
+          :src="config.imageUrl"
+          :srcset="config.imageSrcSet">
+      </div>
+      <!-- Grid lines -->
+      <div
+        :style="{
+          height: 'inherit',
+          width: 'inherit',
+          position: 'absolute',
+          top: 0,
+          left: 0,
+          backgroundImage:
+            'linear-gradient(-90deg, rgba(0, 0, 0, 0.1) 1px, transparent 1px), linear-gradient(rgba(0, 0, 0, 0.1) 1px, transparent 1px)',
+          backgroundSize: `${grid.pitch}px ${grid.pitch}px, ${grid.pitch}px ${grid.pitch}px`,
+          visibility: context.editmode && grid.enable ? 'inherit' : 'hidden',
+        }" />
+      <f7-button
+        v-if="!context.editmode"
+        @click="toggleFullscreen"
+        class="fullscreen-icon"
+        :icon-f7="
+          fullscreen
+            ? 'arrow_down_right_arrow_up_left'
+            : 'arrow_up_left_arrow_down_right'
+        " />
+      <div
+        v-if="context.editmode"
+        style="
+          opacity: 0.3;
+          padding: 4px;
+          position: absolute;
+          width: 100%;
+        ">
+        {{ getCurrentScreenResolution() }}
+        <span v-if="isRetina()"><f7-icon
+          tooltip="Screen resolution shown is the fullscreen resolution for websites. Real screen resolution is bigger."
+          f7="info_circle" /></span>
+      </div>
+      <oh-canvas-item
+        v-for="obj in layout"
+        :key="obj.id"
+        :id="obj.id"
+        :gridEnable="grid.enable"
+        :gridPitch="grid.pitch"
```

Comment:
Normally to specify props in templates the kebab-case is preferred i.e. `grid-enable`, `grid-pitch`, can you please change it. I thought there was an ESLint rule to catch this...